### ISSUE TITLE
Remove hcaptcha

### ISSUE THREAD

**Issue by urthurao** [v0l/void.cat#62]
I have a couple reasons for removing hcaptcha.

1) Hcaptcha does not support accessibility access. It has the option, but to enable it you have to visit hcaptcha's website, create an account, confirm your email address, and download a cookie. This is not accessibility friendly. This is data mining friendly. Accessibility options should be available from the get-go without having to do anything like creating an account or downloading cookies.

2) The prompts are just plain creepy. I'm sure others can agree on this, but the prompts are asking us to train AI for them. I don't want to do this at all, and many of the generated images are just disturbing.

**Comment by v0l** [comment]
Yea i hate completing this every time i need to login
